cfgov/jinja2/v1/index.html

```diff
@@ -174,7 +174,7 @@ <h3 class="h1 m-card_heading">97% of consumers</h3>
                 </section>
             </div>
 
-            <p style="margin-top: 30px;margin-bottom: 0;">Figures are updated quarterly. Last update on 07/20/2017</p>
+            <p style="margin-top: 30px;margin-bottom: 0;">Figures are updated quarterly. Last update on 06/04/2018</p>
```

Comment:
We have [utility classes](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-utilities.less#L205) that will do this instead of inline styles.